specs/airr-schema.yaml

```diff
@@ -2400,9 +2439,10 @@ Node:
         node_id:
             type: string
             description: Identifier for this node, matches id in newick string
+            x-airr:
+                miairr: false
         sequence:
             type: string
             description: DNA sequence of this node
-        meta:
-            type: object #????
-            description: A place for users to add custom info
+            x-airr:
+                miairr: false
```

Comment:
Should this be a required field, since (if I understand correctly) initially there is no link to a `Repertoire` object that will allow recovery of the sequence otherwise?

Reply:
> Should this be a required field

`sequence_id` is required, is that what you are referring to @scharch ?

Or are you saying that `sequence_alignment` should link to a separate `Alignment` object and be required. I was interpreting `sequence_alignment` here to be the actual sequence being stored in the node.

Maybe this was already resolved by @javh's commits?

Reply:
@eharkins The question was whether `sequence` should be required. Further thoughts:
1) The field name should be `sequence_alignment` and it should be defined to match the `germline_alignment` at the `Clone` level. 
2) I think it should _not_ be a required field, as it's easy to imagine this causing an explosion in file sizes. The preferred course should be to extract the `sequence_alignment` from the source `Repertoire`.
3) The above will not work when either (a) the alignment in the `Repertoire` differs from that used to build the tree, maybe because the former was done in the context of a single sequence or (b) for internal nodes, which typically won't be part of any `Repertoire`.

Reply:
1. 
> The field name should be sequence_alignment 

seems like this is already the case here. Go ahead and update the description to reflect

> it should be defined to match the germline_alignment at the Clone level.

2.
> I think it should not be a required field, as it's easy to imagine this causing an explosion in file sizes

sounds good to me. Go ahead and make it not required when you get the chance.
> The preferred course should be to extract the sequence_alignment from the source Repertoire.

makes sense; could one also use the non-required `Clone.sequence_id(s)` field we discussed in [this thread](https://github.com/airr-community/airr-standards/pull/294#discussion_r358451825)?

3. 
> The above will not work when either (a) the alignment in the Repertoire differs from that used to build the tree, maybe because the former was done in the context of a single sequence or (b) for internal nodes, which typically won't be part of any Repertoire.

Does the addition of `Clone.sequence_id(s)` help with this (especially if it can reference sequence objects created in the process of inferring internal nodes which don't belong to a repertoire)?

Reply:
@scharch should I resolve this thread? #302 accounts for 1 & 2 above, but I wasn't sure if 3 was answered here.

Reply:
I think you can close it. 3 was really just an observation of the limitation of 2 (and hence a case where the field should be used despite the default preference against it). Perhaps something for the documentation.

Reply:
Ok, I'll close. Do you have a standard for recording changes introduced by a PR that should be reflected in the documentation?

Reply:
erm @javh? should someone submit a pr on the docs page? I assume we'll need a Clone schema page over there anyway...?

Reply:
I hadn't thought about the docs yet... Yes, at some point we should make a Clone schema page.  Think we can get away with a skeleton for now, similar to the Alignment schema page:
https://raw.githubusercontent.com/airr-community/airr-standards/master/docs/datarep/alignments.rst

Reply:
👍 should I resolve this thread then since it sounds like the issue has been raised / noted or does the merging of this minimal schema version need to wait for complete documentation?

Reply:
I say resolve. We can add the docs after the merge.